substrate/frame/revive/src/vm/mod.rs

```diff
@@ -16,17 +16,17 @@
 // limitations under the License.
 
 //! This module provides a means for executing contracts
-//! represented in wasm.
+//! represented in vm.
```

Comment:
I am not sure if just search and replace in all places cut it. For example here it should say that they are PVM contracts.